packages/core/src/api/user.ts

```diff
@@ -111,7 +111,7 @@ export class UsersAPI {
 	 */
 	public async editCurrentGuildMember(
 		guildId: Snowflake,
-		body: RESTPatchAPIGuildMemberJSONBody = {},
+		body: RESTPatchAPICurrentGuildMemberJSONBody = {},
```

Comment:
Is there no matching return type in discord-api-types for this?

Reply:
This does not return anything different, does it?

Reply:
https://github.com/discordjs/discord-api-types/issues/1367